conf/application.conf

```diff
@@ -105,11 +105,11 @@ microservice {
   services {
     govuk-notify {
       api-key {
-        iss-uuid = no-default-credential
-        secret-key-uuid = no-default-credential
+        iss-uuid = add5f813-2bbc-4855-a88b-3214ed8ad7bb
+        secret-key-uuid = 805c68c6-c04c-4158-95af-95f2d60238ef
       }
       template_id = "dce5ac8a-0970-41a0-b993-bde1beab5825"
-      host = "https://api.notifications.service.gov.uk"
+      host = "http://localhost:6099"
     }
```

Comment:
and this too i suppose

Reply:
by default we need to talk to the stub, hence the change to localhost

Reply:
yes correct